fdbclient/ThreadSafeTransaction.cpp

```diff
@@ -402,10 +402,11 @@ const char* ThreadSafeApi::getClientVersion() {
 }
 
 ThreadFuture<uint64_t> ThreadSafeApi::getServerProtocol(const char* clusterFilePath) {
```

Comment:
We should add some documentation to this function

Reply:
Done